### DISCUSSION TITLE
my-docs-new/zh/codenotes/back-end/docker/Docker

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#819] [Announcements]
# my-docs-new/zh/codenotes/back-end/docker/Docker

Docker实用篇 1.初识Docker 1.1.什么是Docker 微服务虽然具备各种各样的优势，但服务的拆分通用给部署带来了很大的麻烦。 分布式系统中，依赖的组件非常多，不同组件之间部署时往往会产生一些冲突。 在数百上千台服务中重复部署，环境不一定一致，会遇到各种问题 1.1.1.应用部署的环境问题 大型项目组件较多，运行环境也较为复杂，部署时会碰...

https://keaidewenbing.gitee.io/my-docs-new/zh/codenotes/back-end/docker/Docker.html

**Comment by thylove-dsq** [comment]
good
